courses/free-courses-en.md

```diff
@@ -264,6 +264,7 @@
 * [MIT Deep Learning and Artificial Intelligence Lectures](https://deeplearning.mit.edu) - Lex Fridman, et al.
 * [MIT's Artificial Intelligence](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-034-artificial-intelligence-fall-2010/) - Prof. Patrick Henry Winston (MIT OpenCourseWare)
 * [Stanford CS221: Artificial Intelligence: Principles and Techniques \| Autumn 2019](https://www.youtube.com/playlist?list=PLoROMvodv4rO1NB9TD4iUZ3qghGEGtqNX) - Stanford Online
+* [Artificial Intelligence Full Course 2022](https://www.youtube.com/watch?v=lTzHlU3OrXs) - Simplilearn
```

Comment:
We prefer 'playlist' links for YouTube contributions, and be sure to include the full title as provided by the resource